app/src/forms/submission/controller.js

```diff
@@ -18,6 +18,42 @@ module.exports = {
       next(error);
     }
   },
+  getNotes: async (req, res, next) => {
+    try {
+      const response = await service.getNotes(req.params.formSubmissionId);
+      res.status(200).json(response);
+    } catch (error) {
+      next(error);
+    }
+  },
+  addNote: async (req, res, next) => {
+    try {
+      const response = await service.addNote(req.params.formSubmissionId, req.body, req.currentUser);
+      res.status(200).json(response);
+    } catch (error) {
+      next(error);
+    }
+  },
+  getStatus: async (req, res, next) => {
+    try {
+      const response = await service.getStatus(req.params.formSubmissionId);
+      res.status(200).json(response);
+    } catch (error) {
+      next(error);
+    }
+  },
+  addStatus: async (req, res, next) => {
+    try {
+      const response = await service.createStatus(req.params.formSubmissionId, req.body, req.currentUser);
+      // send an email (async in the background)
+      if (req.body.assignmentNotificationEmail) {
+        emailService.statusAssigned(response[0], req.body.assignmentNotificationEmail, req.headers.referer).catch(() => { });
```

Comment:
Maybe... the key is to not have the submission call be delayed or failed on the assignment email call.
If you look down in the `statusAssigned` method it will log anything, so there wouldn't be a missed error log.
I think we're ok to leave this as it is.

Reply:
Ok - let's leave it as is for now - if statusAssigned is catching all throw cases anyways, that catch suppression is there just for safety reasons.